README.md

```diff
@@ -0,0 +1,78 @@
+[![N|Solid](https://renew-wireless.org/figs/cropped-v4-qgkw8d-1024x320.png)](https://renew-wireless.org/)
+
+
+# RENEWLab
+
+
+# Description
+The *RENEWLab* software suite functions as the application layer of the RENEW massive MIMO platform [(official documentation)](https://docs.renew-wireless.org). It provides a user interface through a set of APIs. Users can generate, manipulate, transmit, and receive RF signals on the RENEW hardware by calling these APIs. 
```

Comment:
Let's change this to `RENEWLab is a software toolbox for the RENEW massive MIMO platform...`